app/src/main/java/com/example/cm/ui/meetup/CreateMeetup/CreateMeetupViewModel.java

```diff
@@ -102,7 +101,9 @@ public void setIsPrivate(Boolean isPrivate) {
 
     public void createMeetup() {
         Objects.requireNonNull(selectedUsers.getValue());
-        meetupToAdd = new Meetup(
+        String meetupId = UUID.randomUUID().toString();
```

Comment:
Why create an own ID here? Doesn't Firebase set one itself? Is the ID of the meetup then the same as the Document ID we use to retrieve items?

Reply:
Yes thats the reason. The id in Meetup is annotated with @DocumentId, that sets the id to the documentId